bpf/bpf_lxc.c

```diff
@@ -1067,6 +1067,10 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx, __u32 *d
 			goto skip_egress_gateway;
 
 		if (egress_gw_request_needs_redirect(ip4, &tunnel_endpoint)) {
+			if (tunnel_endpoint == EGRESS_GATEWAY_NO_GATEWAY) {
+				/* Special case for no gateway to drop the traffic */
+				return DROP_HOST_UNREACHABLE;
```

Comment:
Should we define and use a new drop reason (for example `DROP_NO_EGRESS_GATEWAY`) here? That could make it easier to track these drops

(I think in case we just need a couple of new values in `bpf/lib/common.h` and `api/v1/flow/flow.proto`)